lib/households/view/households.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import 'package:gibsonify/navigation/navigation.dart';
+import 'package:gibsonify_repository/gibsonify_repository.dart';
+import 'package:intl/intl.dart';
+
+import '../bloc/households_bloc.dart';
+
+class HouseholdsScreen extends StatelessWidget {
+  const HouseholdsScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) =>
+          HouseholdsBloc(isarRepository: context.read<IsarRepository>())
+            ..add(const HouseholdsPageOpened()),
+      child: const HouseholdsView(),
+    );
+  }
+}
+
+class HouseholdsView extends StatelessWidget {
```

Comment:
can this be inside first class again, especially as there is another widget named very similarly

Reply:
Done